lib/job-iteration/iteration.rb

```diff
@@ -288,6 +261,44 @@ def handle_completed(completed)
       raise "Unexpected thrown value: #{completed.inspect}"
     end
 
+    def cursor_position_from_job_data(job_data)
+      # Backwards compatibility for jobs interrupted before cursor serialization feature shipped
+      return job_data["cursor_position"] unless job_data.key?("serialized_cursor_position")
+
+      deserialize_cursor(job_data.fetch("serialized_cursor_position"))
+    end
+
+    def serialize_cursor(cursor)
+      ActiveJob::Arguments.serialize([cursor]).first
+    rescue ActiveJob::SerializationError
+      raise if job_iteration_enforce_serializable_cursors
```

Comment:
This used to wrap the error in a custom `CursorError`, but I figure it might be adequate to just re-raise the `ActiveJob::SerializationError`.